walk.py

```diff
@@ -0,0 +1,35 @@
+import os
+import re
+import shutil
+import sys
+from utils import *
+
+if os.name != 'nt':
+	print('As of now, only Windows is supported. :(')
+	exit()
+
+if len(sys.argv) < 3:
+	print('Usage: walk.py <root_directory> <target_root_directory>')

```

Comment:
NIT: letting user specify a target root is very troublesome. That means you take the responsibility of checking target exists or not, permission stuff... I would suggest to output everything into a $TEMP folder, output the folder path, and let user take care of the rest.